app/src/main/java/org/sopt/at/login/component/topbar/LoginBackTopBar.kt

```diff
@@ -9,22 +8,19 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.vector.ImageVector
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.vectorResource
 import org.sopt.at.R
 
 @Composable
-fun LoginBackTopBar(modifier: Modifier = Modifier){
-    val context = LocalContext.current
+fun LoginBackTopBar(modifier: Modifier = Modifier, onClick:()->Unit){
```

Comment:
컴포저블의 인자 순서 컨벤션이 있다는 것을 아시나요?
1. 필수 인자
2. 수정자
3. 선택 인자
순으로 작성해주시면 됩니다! 

```suggestion
fun LoginBackTopBar(
    onClick:()->Unit,
    modifier: Modifier = Modifier
) {
```

이런식으로 작성해주시면 됩니다. 
그리고 줄바꿈은 코드 가독성을 높이는 데에 좋습니다! 보통 한 줄에 인자 하나를 작성해주는데요.
현재는 두개 뿐이지만 인자가 늘어난다면 한줄에 하나씩 작성해주시면 좋아요!